src/includes/getting-started-install/apple.macos.mdx

```diff
@@ -1,12 +1,7 @@
 We recommend installing the SDK with CocoaPods. To integrate Sentry into your Xcode project, specify it in your _Podfile_:
 
 ```ruby {filename:Podfile}
-platform :macos, '8.0'
-use_frameworks! # This is important
-
-target 'YourApp' do
-  pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :tag => '{{ packages.version('sentry.cocoa') }}'
-end
+pod 'Sentry', '~> {{ packages.version('sentry.cocoa') }}'
```

Comment:
Also, since we should optimize for beginners, it's good to clarify where the `pod` goes. Given that `target '..' do` is important. When I was trying to add this to Flutter it was helpful since Flutter buries it into their own generated podfile IIRC.